mmv1/templates/terraform/examples/cloud_run_service_secret_volumes.tf.erb

```diff
@@ -1,9 +1,9 @@
 data "google_project" "project" {
-  provider = google-beta
+  provider = google
```

Comment:
I think you probably want to just remove all these lines.

Reply:
Done.